docs/docs/building-and-running.md

```diff
@@ -17,7 +17,7 @@ To push the images and run them on kubernetes, you'll need to install:
 
 -   [GNU tar](https://www.gnu.org/software/tar/) - (Mac only) MacOS ships with `BSD tar`. However, you will need `GNU tar` for docker images operations. On MacOS, you can install `GNU Tar` via [Homebrew](https://brew.sh/): `brew install gnu-tar`
 -   [gcloud](https://cloud.google.com/sdk/gcloud/) - For the `kubectl` tool used to control your Kubernetes cluster. You will also need to this to deploy to our test and production environment on Google Cloud.
--   [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md) to manage kubernetes deployments and config.
+-   [Helm 2](https://helm.sh/docs/intro/install/) to manage kubernetes deployments and config.
```

Comment:
Probably a better doc URL for helm2 :
https://v2.helm.sh/docs/using_helm/#installing-helm

Reply:
Ah, I wasn't sure how to get v2 docs but it's in the banner! Updated.